app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt

```diff
@@ -47,7 +47,7 @@ class MainFragment : Fragment() {
     private fun prepTakePhoto() {
         if(ContextCompat.checkSelfPermission(context!!, Manifest.permission.CAMERA) == PackageManager.PERMISSION_GRANTED){
             takePhoto()
-        }else{
+        } else {
```

Comment:
added horizontal whitespace according to:
https://kotlinlang.org/docs/reference/coding-conventions.html